rubble/src/link/connection.rs

```diff
@@ -532,6 +543,9 @@ enum LlcpError {
 
     /// Consider the connection lost due to a critical error or timeout.
     ConnectionLost,
+
+    /// Shouldn't send Control PDU *LL_VERSION_IND* more than once.
```

Comment:
```suggestion
    /// Already sent Control PDU *LL_VERSION_IND* (indicates invalid behavior by the other device).
```

Reply:
I applied the changes in the follow-up commit :)